src/spec.ts

```diff
@@ -139,6 +139,13 @@ export function normalizeExtendedUnitSpec(spec: ExtendedUnitSpec): Spec {
 }
 
 export function normalizeUnitSpec(spec: UnitSpec): Spec {
+  if (contains([ERRORBAR], spec.mark)) {
+    return normalizeCompositeUnitSpec(spec);
+  }
+  return normalizeRangedUnitSpec(spec);
```

Comment:
If would put the condition to check if this spec is range here -- for consistency.  (so `normalizeRangedUnitSpec` only transform the spec)